roles/api-environment/files/onbase-docs-api/run/Dockerfile

```diff
@@ -1,7 +1,20 @@
-FROM mono:6.6
+FROM mono:latest
 
-WORKDIR /usr/src/$API_NAME
+RUN apt-get update && \
+    apt-get install -y mono-fastcgi-server4 && \
+    apt-get install -y nginx && \
+    touch /run/nginx.pid && \
+    groupadd nginx && \
+    chgrp -R nginx /var/lib/nginx && \
+    chgrp -R nginx /var/log/nginx && \
```

Comment:
I believe this can be combined to

```sh
chgrp -R nginx /var/lib/nginx /var/log/noginx
```

Ref: https://linuxhandbook.com/chgrp-command/